<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 49, 49);
font: 9pt \&quot;Noto Serif\;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: white;
font-size: 16pt;
background-color: none;
border: none;</string>
        </property>
        <property name="text">
         <string>Новый клиент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 16pt;
color: white;
padding-left: 10px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="styleSheet">
         <string notr="true">font-size: 16pt;
color: white;
padding-left: 10px;</string>
        </property>
        <property name="placeholderText">
         <string>Комментарий</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">font-size: 16pt;
color: white;
padding-left: 10px;</string>
        </property>
        <property name="placeholderText">
         <string>От кого</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listView"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: white;
    background-color: rgba(255, 255, 255, 30);
	border: 1px solid rgba(255, 255, 255, 40);
	border-radius: 7px;
	width: 240px;
	height: 50px;
	font-size: 14pt;
}
QPushButton:hover {
	background-color: rgba(255, 255, 255, 40);
}
QPushButton:pressed {
	background-color: rgba(255, 255, 255, 60);
}</string>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
        <property name="icon">
         <iconset resource="static/icons/detail_icon.qrc">
          <normaloff>:/newPrefix/plus-large-svgrepo-com.svg</normaloff>:/newPrefix/plus-large-svgrepo-com.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="static/icons/detail_icon.qrc"/>
 </resources>
 <connections/>
</ui>
